### PR TITLE
feat(ci): publish to winget on release

### PR DESCRIPTION
**Pull request by ST-DDT**
Based on https://github.com/rustdesk/rustdesk/blob/964d4f1f87e656620ac58207c6d62f1084ead6e6/.github/workflows/winget.yml#L9

I'm unable to test this without an actual release.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -106,3 +106,19 @@ jobs:
...
+          fetch-depth: 0
+
+      - name: Get the latest tag
+        run: echo "TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
+
+      - uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: GorillaDevs/Ferium
+          version: ${{ env.TAG }}
+          token: ${{ secrets.COMMITTER_TOKEN }}
```
Comment (ST-DDT) [inline .github/workflows/release.yml:124]: Not sure if this is the correct secret.

**Comment by ST-DDT** [comment]
> please send a signing request to [cla-requests@gdlauncher.com](mailto:cla-requests@gdlauncher.com) and add your github handle to [contributors list](https://github.com/gorilla-devs/clabot-config/blob/master/.clabot?rgh-link-date=2024-06-13T17%3A43%3A53Z).

Do I have to a send an email and add my name to some list?
What should I send to that email?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -106,3 +106,19 @@ jobs:
           tag-name: ${{ env.TAG }}
         env:
           COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
+
+  winget-update:
+    runs-on: ubuntu-latest
```
Comment (theRookieCoder) [inline .github/workflows/release.yml:111]: I know komac has linux and macos binaries, but does this workflow work on linux? The readme uses `windows-latest` in its examples.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -106,3 +106,19 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get the latest tag
```
Comment (theRookieCoder) [inline .github/workflows/release.yml:117]: It's not necessary to calculate this, the `version` field automatically gets the tag and strips the `v` prefix

File: .github/workflows/release.yml (hunk 1 above)
Comment (theRookieCoder) [reply .github/workflows/release.yml:124]: Don't worry I'll edit in the secret var name after merging.

**Review by theRookieCoder — CHANGES_REQUESTED** [review]

**Comment by theRookieCoder** [comment]
> Do I have to a send an email and add my name to some list?

Sorry for the inconvenience, I'll contact @blarfoon about this.

File: .github/workflows/release.yml (hunk 2 above)
Comment (theRookieCoder) [reply .github/workflows/release.yml:111]: Hmm I see I guess it works then. I think linux start up faster on actions anyways so it's probably better.

**Review by theRookieCoder — APPROVED** [review]

**Review by theRookieCoder — APPROVED** [review]

**Comment by ST-DDT** [comment]
@theRookieCoder What is the current status of this?

**Pull request merged**